### PR TITLE
Test OperatorHub & Master on Presubmit hooks

### PR DESCRIPTION
**Pull request by kpouget**

### CODE REVIEW

**Comment by kpouget** [comment]
@ArangoGutierrez PTAL

File: ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master.yaml [hunk 1]
```diff
@@ -40,6 +40,30 @@ tests:
         requests:
           cpu: 2000m
           memory: 2Gi
+    - as: presubmit-operatorhub-cleanup
```
Comment (ArangoGutierrez) [inline ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master.yaml:43]: Clean up? I don't understand this name convention

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**